pkgs/development/python-modules/captcha/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib, fetchPypi, buildPythonPackage, nose, pillow }:
+
+buildPythonPackage rec {
+  pname = "captcha";
+  version = "0.4";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-KuXo2qxPFkm1ezQyi8xFumkbXHB/xvvdAW4hOuzpqLg=";
+  };
+
+  propagatedBuildInputs = [ pillow ];
+
+  nativeCheckInputs = [ nose ];
```

Comment:
Define an explicit `checkPhase`, so we're not relying on `python setup.py test`, which is deprecated.

https://github.com/lepture/captcha/blob/master/tox.ini#L9